README.md

```diff
@@ -1,8 +1,63 @@
+<<<<<<< HEAD
 # JanusGraph Docker images
 
+Repository for building and publishing [JanusGraph][JG] docker images.
+
 ## License
 
 JanusGraph Docker images are provided under the [Apache 2.0
 license](APACHE-2.0.txt) and documentation is provided under the [CC-BY-4.0
 license](CC-BY-4.0.txt). For details about this dual-license structure, please
 see [`LICENSE.txt`](LICENSE.txt).
+
+## Building ##
+
+The `build-images.sh` script will build the docker images for all the Dockerfiles in the versioned
+folder directories.
+
+## Publishing ##
+
+The `push-images.sh` script will push the docker images for all the versioned folders in the repo.
+
+Prior to publishing, you'll need to login to [Docker Hub][DH] using the `docker login` command.
+
+## Configuration ##
+
+The docker containers are configured through environment variables. The environment variables are 
+structured to reflect the different configuration properties available in the 
+[JanusGraph Configuration][JG_CONFIG] .
+
+**Example**
+
+| Configuration Key       | Environment Variable        |
+| ----------------------- | --------------------------- |
+| storage.backend         | JANUS_STORAGE_BACKEND       |
+| storage.cql.keyspace    | JANUS_STORAGE_CQL_KEYSPACE  |
+| index.search.backend    | JANUS_INDEX_SEARCH_BACKEND  |
```

Comment:
I would add one config option that includes a hyphen like `query.force-index` to show how that looks as an environment variable. (I know it's already explained in the description below but it doesn't hurt to also show it as an example I think.)